src/App/Repository/TagRepository.php

```diff
@@ -114,6 +116,40 @@ public function create(Entity $entity)
         return $id;
     }
 
+    public function createMany(Collection $collection) : array
+    {
+        // Check MySQL `innodb_autoinc_lock_mode` = 0 or 1 before running
+        $lockMode = DB::query(Database::SELECT, "SHOW VARIABLES LIKE 'innodb_autoinc_lock_mode'")
```

Comment:
Need to move this to a trait or something

Reply:
Yeah, having this as a generic helper class would be nice.

Reply:
Done